src/store/invitations/models/invitationApart.model.ts

```diff
@@ -0,0 +1,16 @@
+import { AddressModel } from '@/shared/models/address.model';
+import { InvitationApartDTOModel } from '@/store/invitations/models/invitationApartDTO.model';
+
+export class InvitationApartModel {
```

Comment:
Please, do not use shorthands in places, where you can avoid them) 'InvitationApartmentModel' is not a long name)